app/controllers/delegates_controller.rb

```diff
@@ -53,6 +53,7 @@ def create
         format.json { render :show, status: :created, location: @delegate }
       else
         delegates_for_current_profile
+        @staff_list = current_staff_profile.staff_list_json
```

Comment:
Rendering the index (on the next line) requires the `@staff_list` instance variable to be set.